src/www/ui/showjobs.php

```diff
@@ -146,7 +134,7 @@ public function Output()
       $jq_pk = GetParm("jobid",PARM_INTEGER);
       $action = GetParm("action",PARM_STRING);
       $uploadPk = GetParm("upload",PARM_INTEGER);
-      if (!empty($uploadPk) && !$this->uploadDao->isEditable($uploadPk, Auth::getGroupId())){
+      if (!(empty($uploadPk) || $uploadPk === -1) && !$this->uploadDao->isEditable($uploadPk, Auth::getGroupId())){
```

Comment:
Oh, you are right. One might have to check for something like:
- `is_admin || Auth::getGroupId() == creator_group`.

Or are there other privileged users?